### PR TITLE
Appendix g dev debug user directory error

### PR DESCRIPTION
**Pull request by weilixu**
The command line interface is failing when calling create PRM method.
Failing message indicates the directory for user data files are arrays instead of strings.

Fix the issue in the convert_user_data_to_json function, and verified the workflow in the command line.

### CODE REVIEW

**Review by dmaddoxwhite — APPROVED** [review]
Looks good!

File: lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.rb [hunk 1]
```diff
@@ -24,16 +24,18 @@ def convert_userdata_csv_to_json(user_data_path, project_path)
...
     json_objs = {}
-    Dir.glob(src_csv_dir) do |csv_full_name|
+
+    Dir.glob(src_csv_dir).each do |csv_full_name|
       json_rows = []
       csv_file_name = File.basename(csv_full_name, File.extname(csv_full_name))
       json_objs[csv_file_name] = json_rows
     end
 
+
```
Comment (mdahlhausen) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.rb:34]: remove extra whitespace added

File: test/90_1_prm/test_appendix_g_prm.rb [hunk 2]
```diff
@@ -2794,6 +2794,7 @@ def change_bldg_type(model, arguments)
   # @param model [OpenStudio::model::Model] OpenStudio model object
   # @param arguments [Array] List of arguments
   def remove_transformer(model, arguments)
+    assert(model, 'Unsuccessfully generating the model')
```
Comment (mdahlhausen) [inline test/90_1_prm/test_appendix_g_prm.rb:2797]: remove this assertion

**Review by mdahlhausen — CHANGES_REQUESTED** [review]
see comments

**Comment by weilixu** [comment]
> see comments

@mdahlhausen  Updated based on comments

**Review by mdahlhausen — APPROVED** [review]

**Pull request merged**